source/set_up_3dsecure/index.html.md.erb

```diff
@@ -1,56 +1,103 @@
 ---
-title: Set up 3D Secure
+title: Set up Strong Customer Authentication
 last_reviewed_on: 2020-06-10
 review_in: 3 months
 weight: 170
 ---
 
-# Set up 3D Secure (3DS)
+# Strong Customer Authentication
 
-How you set up 3DS payment authentication depends on:
+Strong Customer Authentication (SCA) makes online payments more secure for users and reduces fraud. SCA is part of the Payment Service Directive (PSD2).
```

Comment:
can we link to the product page here if people want to read more about what SCA is, and what it means for services and users?

Reply:
Agreed